packages/react/src/helpers/validateEircode.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  type AddressBase,
+  type AddressSchema,
+  getAddressSchema,
+} from '@farfetch/blackout-client';
+
+/**
+ * Validates eircode and returns true or false depending on the test result..
+ *
+ * @param address - address that will be tested the eircode.
+ *
+ * @returns - boolean which is the result of the validation.
+ * */
+export default async function validateEircode(
+  address: AddressBase,
+): Promise<boolean> {
+  const countryCode = address.country.alpha2Code;
+  const addressSchema: AddressSchema = await getAddressSchema(countryCode);
```

Comment:
I think it is better to remove the call to the client here and accept a parameter with the country address schemas. This will change the return value of the function as well to be a `boolean` instead of a `Promise<boolean>`.